pkg/network/protocols/http/telemetry.go

```diff
@@ -100,3 +100,12 @@ func (t *Telemetry) Log() {
 		aggregations,
 	)
 }
+
+// InternalCounters for testing purpose only
+func (t *Telemetry) InternalCounters() (hits1XX int64, dropped int64, rejected int64, malformed int64) {
+	hits1XX = t.hits1XX.Get()
+	dropped = t.dropped.Get()
+	rejected = t.rejected.Get()
+	malformed = t.malformed.Get()
+	return hits1XX, dropped, rejected, malformed
+}
```

Comment:
```suggestion
func (t *Telemetry) InternalCounters() (int64, int64, int64, int64) {
	hits1XX := t.hits1XX.Get()
	dropped := t.dropped.Get()
	rejected := t.rejected.Get()
	malformed := t.malformed.Get()
	return hits1XX, dropped, rejected, malformed
}
```

Reply:
naming the return value make sense if you look at the api you will understand which fields are return in which place